docs/howto/setup_and_run_vertexing.md

```diff
@@ -0,0 +1,130 @@
+# ACTS Vertexing Tutorial - Example: Adaptive Multi-Vertex Finder (AMVF) - Pythia8
+This tutorial sets up and runs the ACTS Adaptive Multi-Vertex Finder on smeared ACTS Pythia8 truth tracks.
+*Note*: You have to have Pythia8 available on your machine for following this tutorial.
+## Prerequisites
+For setting up and building ACTS, please refer to the general how-to ACTS guide. For this example you will need to enable Pythia8 in the examples by adding
+```bash
+-DACTS_BUILD_EXAMPLES_PYTHIA8=ON
+```
+to your cmake command.
+
+Note: Additionally adding
```

Comment:
Are you sure that this is needed? The default build type should already be `Release`. I would rephrase it to say that it is best to build a `Release` built which is the default. If you have set it to something else, you can use the following settings to set it back to it.

Reply:
Are you sure about this? Every time I build Acts from scratch without specifying this option, the execution is extremely slow afterwards and only after rebuilding with ```DCMAKE_BUILD_TYPE=Release```, it speeds up significantly. I couldn't find where the default DCMAKE_BUILD_TYPE is specified though. So I'm not sure about this.

Reply:
You are right. I checked again and without an explicit build type no additional build flags are set. 

It would be a good idea to set a default build type e.g. Release if none is given. I'll open an issue for tha.